kobo/apps/stripe/tests/test_organization_usage.py

```diff
@@ -438,31 +438,23 @@ def test_user_not_member_of_organization(self):
         assert response.status_code == status.HTTP_400_BAD_REQUEST
 
     def test_successful_retrieval(self):
-        generate_enterprise_subscription(self.organization)
+        generate_mmo_subscription(self.organization)
         create_mock_assets([self.anotheruser])
         response = self.client.get(self.detail_url)
         assert response.status_code == status.HTTP_200_OK
         assert response.data['count'] == 1
         assert response.data['results'][0]['asset__name'] == 'test'
         assert response.data['results'][0]['deployment_status'] == 'deployed'
 
-    def test_aggregates_usage_for_enterprise_org(self):
-        generate_enterprise_subscription(self.organization)
+    def test_aggregates_usage_for_mmo(self):
+        generate_mmo_subscription(self.organization)
         self.organization.add_user(self.newuser)
         # create 2 additional assets, one per user
         create_mock_assets([self.anotheruser, self.newuser])
         response = self.client.get(self.detail_url)
         assert response.status_code == status.HTTP_200_OK
         assert response.data['count'] == 2
 
-    def test_users_without_enterprise_see_only_their_usage(self):
```

Comment:
With the introduction of the MMO feature, it is no longer possible to have an organization with multiple users that doesn't aggregate usage. Therefore, this test is irrelevant.